packages/react-core/src/components/SimpleList/SimpleList.tsx

```diff
@@ -14,6 +14,8 @@ export interface SimpleListProps extends Omit<React.HTMLProps<HTMLDivElement>, '
     ref: React.RefObject<HTMLButtonElement> | React.RefObject<HTMLAnchorElement>,
     props: SimpleListItemProps
   ) => void;
+  /** Indicates whether component is controlled by its internal state (by default it's true) */
```

Comment:
You can remove the "(by default it's true)". The documentation lists the default values.

Reply:
ok